unity-renderer/Assets/Rendering/Utils/GPUSkinning/SimpleGPUSkinning.cs

```diff
@@ -21,28 +23,59 @@ public static void EncodeBindPosesIntoMesh(SkinnedMeshRenderer skr)
         {
             Mesh sharedMesh = skr.sharedMesh;
             int vertexCount = sharedMesh.vertexCount;
-            Vector4[] bone01data = new Vector4[vertexCount];
-            Vector4[] bone23data = new Vector4[vertexCount];
-
-            BoneWeight[] boneWeights = sharedMesh.boneWeights;
+            
+            NativeArray<Vector4> bone01data = new NativeArray<Vector4>(vertexCount, Allocator.Temp, NativeArrayOptions.UninitializedMemory);
```

Comment:
you are not disposing the arrays.. could this be a reason of memory leaks?

Reply:
Doesn't it require a `using` to perform the disposal properly?

Reply:
I did a test for the sake of it, I've changed the allocation of the native array to Persistent and didn't dispose them, after a few frames Unity actually tells you that is leaking 
![image](https://user-images.githubusercontent.com/7646450/173662762-87420e4d-811a-46df-8f65-df436fa55a15.png)

So calling dispose or not with Allocation.Temp is the same, unity disposes it when the reference count is zero after a few frames.

Reply:
I will add the Disposes to avoid future confusion.